### ISSUE TITLE
version without qwiic switch for GauteH

### ISSUE THREAD

**Issue by jerabaul29** [jerabaul29/OpenMetBuoy-v2021a#98]
@gauteh which firmware version do you use? :) Do you usually use a turn key binary from https://github.com/jerabaul29/OpenMetBuoy-v2021a/tree/main/legacy_firmware/binaries (if so, which one? I guess one of the steval ones? :) ), or do you recompile? :)

I will adapt the corresponding firmware to use without qwiic switch, with pin D43 to provide power to the IMU :) .

**Comment by gauteh** [comment]
We need to know which pins to connect to sda,scl. Since the qwiic cable is necessary, there is no use in just wiring across the switch-footprint.

**Comment by gauteh** [comment]
Yes, that's perfect. We will try it out (using headers @trygvesovik so that we don't loose any AGT's).

**Comment by trygvesovik** [comment]
Hello, when testing the new circuit board without qwiic switch, we received the error message "failed to start GNSS reboot". It was working until we connected the wires between SCL and SDA, so it seems like the problem is related to this. It also seemed to work with the qwiic switch, when using the old firmware, so all the components should be fine.

**Comment by gauteh** [comment]
I was wondering if there could be some I2C port collision? Or maybe the IMU is not turned properly off and thus causing trouble for the GPS?

**Comment by tnn77** [comment]
Hi, @jerabaul29.
 
I searched back to when you helped us to be able to use the PCBs instead of the Qwiic cables (you can follow the below link to see the fix you did).   
Hope this helps. 

https://github.com/jerabaul29/tracker_and_waves_instrument/issues/116#issuecomment-944190199

**Comment by trygvesovik** [comment]
@askbreivik

**Comment by tnn77** [comment]
Great, thank you so much, @jerabaul29
